lib/archethic/utils/regression/benchmarks/tps.ex

```diff
@@ -0,0 +1,98 @@
+defmodule ArchEthic.Utils.Regression.Benchmark.TPS do
+  @moduledoc """
+  Module for regession testing the paralleing processing of transactions and
+  benchmarking the parallel transaction processing capability of a node
+  """
+
+  require Logger
+
+  alias ArchEthic.Utils.Regression.Benchmark
+  alias ArchEthic.Utils.Regression.Playbook
+  alias ArchEthic.Crypto
+  alias ArchEthic.Utils.Regression.Benchmarks.Helpers.TPSHelper
+
+  alias ArchEthic.TransactionChain.TransactionData
+  alias ArchEthic.TransactionChain.TransactionData.Ledger
+  alias ArchEthic.TransactionChain.TransactionData.UCOLedger
+  alias ArchEthic.TransactionChain.TransactionData.UCOLedger.Transfer, as: UCOTransfer
+
+  @behaviour Benchmark
+
+  def plan([host | _nodes], _opts) do
+    port = Application.get_env(:archethic, ArchEthic.P2P.Listener)[:port]
+
+    {alice, bob} = preliminaries()
+
+    TPSHelper.withdraw_uco_from_testnet_endpoint(alice.address)
+    TPSHelper.withdraw_uco_from_testnet_endpoint(bob.address)
+
+    dispatch_transactions(alice, bob, host, port)
+  end
+
+  def preliminaries() do
+    alice = %{seed: "StupidAlice"}
+    bob = %{seed: "IdiotBob"}
+
+    alice = Map.put(alice, :address, get_address(alice.seed))
+    bob = Map.put(bob, :address, get_address(bob.seed))
+
+    alice = Map.put(alice, :transaction_data, get_transaction_data(bob.address))
+    bob = Map.put(bob, :transaction_data, get_transaction_data(alice.address))
+
+    {alice, bob}
+  end
+
+  def get_address(seed) do
+    {next_public_key, _next_private_key} =
+      Crypto.derive_keypair(seed, 0 + 1, Crypto.default_curve())
```

Comment:
I don't get the 0+1. Please either use 0 or derive last_index for better code readability.

Reply:
It is also possible to derive last_address easily.